packages/common/src/validations/get_specific_contacts_schema.ts

```diff
@@ -0,0 +1,6 @@
+import * as yup from "yup";
+
+export const getSpecificContactsSchema = yup.object({
```

Comment:
Update file name too

```suggestion
export const getContactSchema = yup.object({
```

Reply:
There is already a file called that for when we get all contacts.

Reply:
`getContactSchema` != `getContactsSchema`